ui/widget.js

```diff
@@ -390,6 +408,55 @@ $.Widget.prototype = {
 		return this._setOptions({ disabled: true });
 	},
 
+	_classes: function( element, keys, extra, add, object ) {
```

Comment:
Object makes sense to me. Just as a note, this is not intended to be a documented function. It should not be used directly outside the widget factory itself.